bertopic/_save_utils.py

```diff
@@ -284,6 +285,13 @@ def generate_readme(model, repo_id: str):
     model_card = model_card.replace("{HYPERPARAMS}", params)
     model_card = model_card.replace("{FRAMEWORKS}", frameworks)
     
+    # Fill Pipeline tag
+    has_visual_aspect = check_has_visual_aspect(model)
+    if not has_visual_aspect:
+        model_card = model_card.replace("{PIPELINE_TAG}", "text-classification")
+    else:
+        model_card = model_card.replace("pipeline_tag: {PIPELINE_TAG} /n","") # TODO add proper tag for this instance 
```

Comment:
I will aim to find a way of implementing this properly

Reply:
@davanstrien Great, should we save this for later? If so, then I will go ahead and merge this PR.

Reply:
I think that makes sense for now since this will probably require a bit more work.